File: UserProfileService/UserProfileService.Application/DTOs/DayResultDTOs/CreateDayResultDTO.cs
namespace UserProfileService.Application.DTOs
{
    public class CreateDayResultDTO
    {
        public int ProfileId { get; set; }
        public DateOnly Date { get; set; }
        public double? Weight { get; set; }
        public int GlassesOfWater { get; set; } = 0;
    }
}

File: NutritionBalance.Client/src/app/models/profile/DTOs/dayResult/create-day-result.dto.ts
import { ActivityLevel } from '../../enums/activity-level.enum';

export class CreateDayResultDTO {
  profileId!: number;
  date!: Date;
  weight?: number;
  height?: number;
  activityLevel?: ActivityLevel;
  glassesOfWater: number = 0;
}
